src/qibo/transpiler/decompositions.py

```diff
@@ -83,7 +80,7 @@ def _u3_to_gpi2(t, p, l):
 gpi2_dec.add(gates.SX, [gates.GPI2(0, 0)])
 gpi2_dec.add(
     gates.RX,
-    lambda gate: [
+    lambda gate, backend: [
```

Comment:
Why is `backend` needed if it is never used? This goes for all the other `lambda` functions in this file.